plainly-plugin/src/ui/components/form/ExportForm.tsx

```diff
@@ -1,11 +1,11 @@
+import path from 'path';
```

Comment:
does not work anymore:

![image](https://github.com/user-attachments/assets/18e24afc-37f4-4050-bc38-2667c97c5380)

Reply:
Setting api key also does not work anymore, I have a valid key

![image](https://github.com/user-attachments/assets/2c8706dc-8df9-4fea-bd46-4d49e2acbe22)